### DISCUSSION TITLE
Can't pay overdue payment

### DISCUSSION THREAD

**Discussion by sbkobaidze** [vercel/community#5564] [Account Issue] [answered]
### Page to Investigate

/~/settings/billing

### Steps to Reproduce

I have overdue billing for pro account. When I try to click " Charge now" it says "There are no payments that can be retried."
<img width="503" alt="Screenshot 2024-01-23 013522" src="https://github.com/vercel/community/assets/103885846/12118d5c-f5cd-4528-88ff-3f0d0baa2145">

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out about this. Based on the error you see, it sounds like there is a problem with your current payment method. You may be able to solve it by using a different payment method.

If that doesn't solve it for you, please [submit a case](https://vercel.com/help#issues) to the Customer Success Team. If you select `Invoice Enquiry` as the Problem Area on the form then it will send your ticket to our billing specialists who can help you with this 🙇
